lib/walker.js

```diff
@@ -636,16 +774,35 @@ class Walker {
     // eslint-disable-line camelcase
     if (record[STORE_STAT]) return;
 
+    const realPath = toNormalizedRealPath(record.file);
+    if (realPath !== record.file) {
+      this.appendStat({
+        file: realPath,
+        store: STORE_STAT,
+      });
+    }
+
     try {
-      record[STORE_STAT] = await fs.stat(record.file);
+      const valueStat = await fs.stat(record.file);
+
+      const value = {
+        mode: valueStat.mode,
+        size: valueStat.isFile() ? valueStat.size : 0,
+        //
```

Comment:
Empty comment line

Reply:
call it a separator between raw properties and calculated properties.

Reply:
Oh ok so maybe add a comment to say that otherwise it seems something is missing

Reply:
I deleted the line, it was adding little purpose anyway